doc/source/cluster/kubernetes/user-guides.md

```diff
@@ -14,6 +14,8 @@ user-guides/storage
 user-guides/config
 user-guides/configuring-autoscaling
 user-guides/kuberay-gcs-ft
+user-guides/kuberay-gcs-ft
```

Comment:
```suggestion
```
Is this line duplicated with the line above it?

Reply:
Yes, fixed.